### ISSUE TITLE
set path for host in config file

### ISSUE THREAD

**Issue by chrismaster** [FriedrichFroebel/brother_ql_web#19]
Is there a way to set the path for the host
eg:
now
http://localhost:8013/labeldesigner
with path
http://localhost:8013/somepath/labeldesigner

the url labeldesigner does not care about the path, but the static files like css
        /static/css/bootstrap.min.css
        /static/css/bootstrap-icons.min.css
        /static/css/custom.css

**Comment by FriedrichFroebel** [comment]
No, there currently is no way to do so (and AFAIK has never been). But I understand that this might be desirable, as the static references might lead to conflicts otherwise when a setup with a reverse proxy or similar is used.

Feel free to submit a corresponding PR (with tests) to handle this.
